vpc.tf

```diff
@@ -47,12 +47,12 @@ module "vpc" {
   private_subnets = local.private_subnets
 
   public_subnet_tags = {
-    "kubernetes.io/cluster/${module.label.id}" = "shared"
-    "kubernetes.io/role/elb"                   = "1"
+    "kubernetes.io/cluster/${local.id}" = "shared"
+    "kubernetes.io/role/elb"            = "1"
   }
 
   private_subnet_tags = {
-    "kubernetes.io/cluster/${module.label.id}" = "shared"
-    "kubernetes.io/role/internal-elb"          = "1"
+    "kubernetes.io/cluster/${local.id}" = "shared"
+    "kubernetes.io/role/internal-elb"   = "1"
```

Comment:
These are generating the tags correctly, right?

Reply:
Yeah this is a placeholder. I want to run it in terragrunt with k8s first. 

That is something I need input on later.